client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -390,6 +390,37 @@ Undocumented.prototype.settings = function( siteId, method = 'get', data = {}, f
 	return this.wpcom.req.post( { path }, { apiVersion }, body, fn );
 };
 
+/**
+ * GET/POST site settings
```

Comment:
I suspect we want to mention that this function relates to keyring settings.

Reply:
Actually it would be great to explain the difference with other keyring-related endpoints, especially for developers outside of our team.